backend/src/migrations/20240825181542_create_files_table.ts

```diff
@@ -0,0 +1,38 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'files';
+
+const ColumnName = {
+    ID: 'id',
+    URL: 'url',
+    TYPE: 'type',
+    CREATED_AT: 'created_at',
+    UPDATED_AT: 'updated_at',
+};
+
+async function up(knex: Knex): Promise<void> {
+    await knex.schema.raw('CREATE EXTENSION IF NOT EXISTS "uuid-ossp";');
+    await knex.schema.createTable(TABLE_NAME, (table) => {
+        table
+            .uuid(ColumnName.ID)
+            .notNullable()
+            .primary()
+            .defaultTo(knex.raw('uuid_generate_v4()'));
+        table.string(ColumnName.URL).notNullable();
+        table.enum(ColumnName.TYPE, ['video', 'photo']).notNullable();
```

Comment:
here it's better to use enu, because enum is a JS reserved word, also we can get advantage of native postgres enum type, passing an optional argument indication to use native type and type name.